narwhal/config/src/lib.rs

```diff
@@ -147,6 +148,8 @@ pub struct Parameters {
     pub prometheus_metrics: PrometheusMetricsParameters,
     /// Network admin server ports for primary & worker.
     pub network_admin_server: NetworkAdminServerParameters,
+    /// Anemo network settings.
+    pub anemo: AnemoParameters,
```

Comment:
Can we have a default initialization (`#[serde(default = "...")]`), so the field does not have to be specified? Otherwise, a couple of config files may need to get updated in the ops repo and deployment would be temporarily broken.

Reply:
Thanks for this suggestion! Done